usagov/vote-gov-nvrf-app
# Vote-723: Cypress Test Update

Ticket - [Vote-723](https://cm-jira.usa.gov/browse/VOTE-723)
Purpose - Updates to cypress to match element names within the app
Testing - 
1. Check that all tests are passing on the run 
2. Pull down locally and run `npm runs start` and then spilt terminal and cd into `testing` and run `npm run cy:test` and check that all tests are passing
